extensions/amp-sidebar/1.0/component.js

```diff
@@ -191,44 +191,49 @@ export function SidebarToolbar({
   ...rest
 }) {
   const ref = useRef();
-  const [matches, setMatches] = useState(false);
-  const [targetEl, setTargetEl] = useState(null);
+  const [target, setTarget] = useState();
```

Comment:
```suggestion
  const [target, setTarget] = useState(null);
```

Reply:
Updated in the latest Revision!  https://github.com/ampproject/amphtml/pull/34158/commits/d1c5a3eb1d7faf164c13ae9c7b13a0b2ac9dc0ed

Reply:
Am I missing something? Still not seeing it

Reply:
Oops, thanks!  I think I got it right this time!